modules/assim.batch/vignettes/MultiSitePDAVignette.Rmd

```diff
@@ -0,0 +1,646 @@
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_0-767/AMF_US-MMS_BASE_HR_8-1.2001-01-01.2014-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.3>
+  <settings.4>
+   <site>
+    <id>768</id>
+    <met.start>2005/01/01</met.start>
+    <met.end>2015/12/31</met.end>
+    <name>Missouri Ozark Site/BREA (US-MOz)</name>
+    <lat>38.7441</lat>
+    <lon>-92.2</lon>
+   </site>
+   <start.date>2005/01/01</start.date>
+   <end.date>2015/12/31</end.date>
+   <inputs>
+    <met>
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_0-768/AMF_US-MOz_BASE_HH_7-1.2005-01-01.2015-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.4>
+  <settings.5>
+   <site>
+    <id>776</id>
+    <met.start>2007/01/01</met.start>
+    <met.end>2014/12/31</met.end>
+    <name>Univiversity of Michigan Biological Station (US-UMB)</name>
+    <lat>45.5598</lat>
+    <lon>-84.7138</lon>
+   </site>
+   <start.date>2007/01/01</start.date>
+   <end.date>2014/12/31</end.date>
+   <inputs>
+    <met>
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_0-776/AMF_US-UMB_BASE_HH_10-1.2007-01-01.2014-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.5>
+  <settings.6>
+   <site>
+    <id>676</id>
+    <met.start>1999/01/01</met.start>
+    <met.end>2006/12/31</met.end>
+    <name>Willow Creek (US-WCr)</name>
+    <lat>45.805925</lat>
+    <lon>-90.07961</lon>
+   </site>
+   <start.date>1999/01/01</start.date>
+   <end.date>2006/12/31</end.date>
+   <inputs>
+    <met>
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_0-676/AMF_US-WCr_BASE_HH_11-1.1999-01-01.2006-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.6>
+  <settings.7>
+   <site>
+    <id>1000000109</id>
+    <met.start>2013/01/01</met.start>
+    <met.end>2015/12/31</met.end>
+    <name>Ontario - Turkey Point Mature Deciduous (CA-TPD)</name>
+    <lat>42.6353</lat>
+    <lon>-80.5577</lon>
+   </site>
+   <start.date>2013/01/01</start.date>
+   <end.date>2015/12/31</end.date>
+   <inputs>
+    <met>
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_1-109/AMF_CA-TPD_BASE_HH_1-1.2013-01-01.2015-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.7>
+  <settings.8>
+   <site>
+    <id>740</id>
+    <met.start>1997/01/01</met.start>
+    <met.end>2010/12/31</met.end>
+    <name>BOREAS SSA Old Aspen (CA-Oas)</name>
+    <lat>53.6289</lat>
+    <lon>-106.198</lon>
+   </site>
+   <start.date>1997/01/01</start.date>
+   <end.date>2010/12/31</end.date>
+   <inputs>
+    <met>
+     <path>/fs/data1/pecan.data/dbfiles/AmerifluxLBL_SIPNET_site_0-740/AMF_CA-Oas_BASE_HH_1-1.1997-01-01.2010-12-31.clim</path>
+    </met>
+   </inputs>
+  </settings.8>
+ </run>
+```
+
+If requested, your `<ensemble>` and `<sensitivity.analysis>` sections would look similar to this in your `pecan.CONFIGS.xml`:
+
+
+```
+ <ensemble>
+  <settings.1>
+   <size>100</size>
+   <variable>NEE</variable>
+   <ensemble.id>1000017326</ensemble.id>
```

Comment:
Is this really necessary? I've never specified ensemble.id within the ensemble block before and I've never specified anything nested within settings tags. Why can't you just use the standard notation and provide a single size and variable tag and let it apply those to all sites using the default run start/end dates. It seems like you only need to do this if the start/end are different from the run's

Reply:
I haven't specified these myself either, workflow produce them automatically? I'm just letting users know what it looks like in the CONFIGS.xml at this point. Otherwise my original pecan.xml has this only:
```
 <ensemble>
  <size>100</size>
  <variable>NEE</variable>
 </ensemble>
```

Reply:
Got it. It's probably good to have the vignette focus on what you put in originally, otherwise it leaves the impression that the user has to do a LOT more work than they actually do.